web/download_manager.js

```diff
@@ -86,20 +86,20 @@ if (typeof PDFJSDev === 'undefined' || PDFJSDev.test('GENERIC || CHROME')) {
     },
 
     download: function DownloadManager_download(blob, url, filename) {
-      if (!URL) {
```

Comment:
Sure, I'll change it to that.

I was concerned that it could change the IE10 / IE11 behavior if I checked `pdfjsLib.PDFJS.disableCreateObjectURL` before checking `navigator.msSaveBlob`.